app/views/welcome/index.html.haml

```diff
@@ -16,7 +16,7 @@
     .root-landing-recovery-stories
       .container
         #recovery-stories-container.extras-hidden
-          = render @recovery_displays
+          = render RecoveryDisplay.limit(5)
```

Comment:
MVC nit: In general, having controller code in templates is an undesirable pattern to imitate — templates are happier the dumber they are.

As a smell of the same: this seems unidiomatic for Rails. See, e.g.:

```erb
<% @products.each do |product| %>
  <% cache product do %>
    <%= render product %>
  <% end %>
<% end %>
```
https://edgeguides.rubyonrails.org/caching_with_rails.html
https://blog.appsignal.com/2018/04/03/russian-doll-caching-in-rails.html
https://medium.com/@adrdilauro/master-html-russian-doll-caching-to-cut-server-response-time-e74cced05247

Reply:
Fine